pages/dao/[symbol]/proposal/components/instructions/Friktion/Deposit.tsx

```diff
@@ -0,0 +1,167 @@
+import * as yup from 'yup';
+import { PublicKey } from '@solana/web3.js';
+import Input from '@components/inputs/Input';
+import Select from '@components/inputs/Select';
+import useFriktionVolt from '@hooks/usefriktionVolts';
+import useGovernanceUnderlyingTokenAccounts from '@hooks/useGovernanceUnderlyingTokenAccounts';
+import useInstructionFormBuilder from '@hooks/useInstructionFormBuilder';
+import { VoltData } from '@tools/sdk/friktion/friktion';
+import depositToVolt from '@tools/sdk/friktion/instructions/depositToVault';
+import { GovernedMultiTypeAccount } from '@utils/tokens';
+import { FriktionDepositForm } from '@utils/uiTypes/proposalCreationTypes';
+import SelectOptionDetailed, {
+  Flag,
+} from '../../../../../../../components/SelectOptionDetailed';
+import TokenAccountSelect from '../../TokenAccountSelect';
+
+const schema = yup.object().shape({
+  governedAccount: yup.object().required('Governance is required'),
+  sourceAccount: yup.string().typeError('Source account is required'),
+  volt: yup.string().required('Volt is required'),
+  uiAmount: yup.number().typeError('Amount is required'),
```

Comment:
TypeError seems like a good thing to ensure its a number, good idea. Should you add `.required()` also?